src/anyp/PortCfg.cc

```diff
@@ -25,13 +27,27 @@ AnyP::PortCfgPointer FtpPortList;
 int NHttpSockets = 0;
 int HttpSockets[MAXTCPLISTENPORTS];
 
-AnyP::PortCfg::PortCfg(const TrafficModeFlags::PortKind aPortKind):
+static AnyP::TrafficModeFlags::PortKind
+ParseDirective(const SBuf &directive)
+{
+    if (directive.cmp("http_port") == 0)
+        return AnyP::TrafficModeFlags::httpPort;
+    else if (directive.cmp("https_port") == 0)
+        return AnyP::TrafficModeFlags::httpsPort;
+    else {
+        assert(directive.cmp("ftp_port") == 0);
+        return AnyP::TrafficModeFlags::ftpPort;
+    }
+}
+
+AnyP::PortCfg::PortCfg(const SBuf &directiveName):
     next(),
     s(),
+    directive(directiveName.cmp("ascii_port") == 0 ? SBuf("http_port") : directiveName),
```

Comment:
> will anything break

Nothing will break, of course. I just thought that printing 'Accepting connections at https_port ...' in both cases would be better.

Reply:
> I just thought that printing 'Accepting connections at https_port ...' in both cases would be better.

It is a matter of opinion, but think that using admin's preferred terminology in for-admin output is better. Or, to put it another way, when reproducing admin's configuration, all other factors being equal, we should be reproducing it as close as possible to the original rather than "upgrading" it in the process.

Said that, I do not want to complicate things. If it were overall easier for us to store modern spelling, I would advocate for that. In this case, storing the original spelling (and adding an `if` statement to the string-to-ID mapping code) is as easy as storing the modern spelling (and adding this tertiary operator). Thus, I would go for preserving the original spelling.

I am resolving this change request but may ask you to preserve the original spelling later.

P.S. Frankly, I would be really surprised if somebody is using `ascii_port` these days. We should remove that ancient alias! Such removal is out of scope in this PR, and the choice in the change request does not affect the ease of that future removal. This change request is a good illustration of why keeping these aliases forever has downsides/costs.